es-vue-base/src/lib-components/EsReview.vue

```diff
@@ -190,18 +330,40 @@ export default {
             return this.userId.toString() === this.reviewerId.toString();
         },
     },
+    methods: {
+        localeDate(date) {
+            // Displays DD/MM/YYYY
+            return new Date(date).toLocaleDateString();
+        },
+    },
 };
 </script>
 
 <style lang="scss" scoped>
 @import '~@energysage/es-bs-base/scss/includes';
 
 .review-holder {
-    .name-holder {
-        max-width: 90%;
+    .details-holder {
+        max-width: 70%;
     }
 }
 
+.title {
+    @include media-breakpoint-down(md) {
+        font-size: $font-size-base;
+    }
+}
+
+.logo {
+    border-radius: 8px;
+    height: 64px;
+    object-fit: contain;
+    width: 64px;
+    @include media-breakpoint-down(md) {
+        height: 32px;
+        width: 32px;
+    }
+}
```

Comment:
I think there's a cleaner way to accomplish this. For `border-radius` you should be able to use a [rounded](https://github.com/EnergySage/es-ds/blob/7d2bff97201a2d4434a22286971ec0dc58e5058d/es-bs-base/scss/utilities/_borders.scss#L33) class.

For the scaling down of the image take a look at the [fluid](https://bootstrap-vue.org/docs/components/image#component-reference) prop on `b-img`

Reply:
I updated the `border-radius` but couldn't get `fluid` to scale the way I expected... 
For testing, I'm using an image that is quite large but can we assume that the uploaded logo will always be the same size?

Reply:
It is not safe the assume unfortunately. We suggest `120x120` but allow them to upload whatever size they want.

Reply:
Gotcha! The dimensions aren't perfect but I got as close as I could in with my most recent changes in [458f10d](https://github.com/EnergySage/es-ds/pull/455/commits/458f10d0ee55badc409b499bf7e2e643266d8dda)